content/patterns/button/examples/button.html

```diff
@@ -126,8 +127,8 @@ <h2 id="rps_label">Role, Property, State, and Tabindex Attributes</h2>
               <td><code>div</code>, <code>a</code></td>
               <td>
                 <ul>
-                  <li>Identifies the element as a <code>button</code> widget.</li>
-                  <li>Accessible name for the button is defined by the text content of the element.</li>
+                  <li>要素を <code>button</code> ウィジェットとして識別します。</li>
+                  <li>ボタンのアクセシブルな名前は、要素のテキストコンテンツによって定義されます。</li>
```

Comment:
このセクションはリストで、言い切りの形にするとより馴染むかなと思います！
```suggestion
                  <li>要素を <code>button</code> ウィジェットとして識別する。</li>
                  <li>ボタンのアクセシブルな名前は、要素のテキストコンテンツによって定義される。</li>
```